plugin/src/main/groovy/com/novoda/gradle/command/Device.groovy

```diff
@@ -70,6 +70,6 @@ class Device {
     }
 
     String toString() {
-        getClass().name + [id: id, sdk: sdkVersion(), version: androidVersion(), brand: brand(), manufacturer: manufacturer(), model: model(), country: country(), language: language(), timezone: timezone()]
+        getClass().simpleName + [id: id, sdk: sdkVersion(), version: androidVersion(), brand: brand(), manufacturer: manufacturer(), model: model(), country: country(), language: language(), timezone: timezone()]
```

Comment:
Also decided to use `simpleName` here. The name was too verbose. Now it just says `Device` which makes sense.